specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2021-03-01/redisenterprise.json

```diff
@@ -0,0 +1,1643 @@
+          "description": "The current primary key that clients can use to authenticate"
+        },
+        "secondaryKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current secondary key that clients can use to authenticate"
+        }
+      }
+    },
+    "RegenerateKeyParameters": {
+      "title": "Regenerate access keys request",
+      "properties": {
+        "keyType": {
+          "type": "string",
+          "description": "Which access key to regenerate.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "x-ms-enum": {
+            "name": "AccessKeyType",
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "keyType"
+      ],
+      "description": "Specifies which access keys to reset to a new random value."
+    },
+    "ImportClusterParameters": {
+      "title": "Import an RDB file into a target database",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "SAS Uri for the target blob to import from"
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise import operation."
+    },
+    "ExportClusterParameters": {
+      "title": "Export an RDB file into a target database",
+      "properties": {
+        "sasUri": {
+          "type": "string",
+          "description": "SAS Uri for the target directory to export to"
+        }
+      },
+      "required": [
+        "sasUri"
+      ],
+      "description": "Parameters for a Redis Enterprise export operation."
+    },
+    "OperationStatus": {
+      "description": "The status of a long-running operation.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation's unique id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation's name."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time of the operation."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The end time of the operation."
+        },
+        "status": {
+          "type": "string",
+          "description": "The current status of the operation."
+        },
+        "error": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse",
+          "description": "Error response describing why the operation failed."
+        }
+      }
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Gets subscription credentials which uniquely identify the Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
```

Comment:
(nit, Is it really called credentials? :/ Its not what I think of as credentials.)